lib/dor/services/client/release_tags.rb

```diff
@@ -40,7 +40,7 @@ def list
 
           raise_exception_based_on_response!(resp, object_identifier) unless resp.success?
 
```

Comment:
Param comment for this method needs to be updated.

Reply:
Done.